src/room.ts

```diff
@@ -374,4 +417,28 @@ export class Room {
         break;
     }
   }
+
+  private async startScreenShare() {
+    let captureStream = null;
+    const options = {
+      video: true,
+    };
+
+    try {
+      captureStream = await navigator.mediaDevices.getDisplayMedia(options);
```

Comment:
what does this do? I usually just call startScreenShare so just curious

Reply:
This allows us to grab the MediaStream manually with our own constraints, which Dominic mentioned [on Slack](https://pluot.slack.com/archives/C01M0U5TZA6/p1646408198877319?thread_ts=1645204505.791569&cid=C01M0U5TZA6) is a common use case with spatialization customers, so we figured it might be good to showcase how to do that here.

Reply:
very cool, thanks!